### ISSUE TITLE
feature: more commands

### ISSUE THREAD

**Issue by Zarkness2** [Sv3ks/HyperCurrencies#1]
```
add a /balance and /bal command to show you the money in all the economies you create

add a command /balancetop <economy name> and /baltop <economy name>.

add the command /pay <user> <economy> <amount> so that if you have the permission hypercurrencies.pay.<economy name> you can pay with that economy otherwise you can only pay with the economy that is set as vault.

add a command to /paytoggle <economy> and /paytoggle all to toggle that you can't send money from any economy

add a command to set an economy as vault.

each command would have its respective permission

add support for mysql/mariadb database

add different placeholders such as 

%hypercurrencie_balance_<currencie>% indicate balance
%hypercurrencie_balance_balance_value_<currencie>% indicate balance value
%hypercurrencie_balance_balance_formatted_<currencie>% indicate formatted balance
%hypercurrencie_top_player_#_<currencie>% indicate the player name of the ranking #
%hypercurrencie_top_balance_#_<currencie>% indicate the player balance of ranking #
%hypercurrencie_top_balance_value_#_<currencie>% indicate the player balance value of ranking #
%hypercurrencie_sum_balance_<currencie>% indicate the balance of server total
%hypercurrencie_sum_balance_balance_value_<currencie>% indicate the balance balance value of server total
%hypercurrencie_top_rank_<currencie>% indicate the rank of the player
%hypercurrencie_top_rank_#_<currencie>% indicate the rank of the player# %hypercurrencie_top_rank_#_<currencie>%
indicate the rank of the player#
%hypercurrencie_global_paypermission_<currencie>% show global pay permission
%hypercurrencie_global_paypermission_<currencie>% show the player pay permission
%hypercurrencie_paytoggle_<currencie>% show the player's accepting payments status

**Comment by Sv3ks** [comment]
Vault integration is already implemented. Simply set the currency's provider to 'vault'. I will add the commands you suggested in 1.3. Also, in the 1.2 snapshot, SQL integration was implemented, but it contains bugs that i will fix in the full 1.2 release. Lastly, thank you for helping out with the plugin.

**Comment by Zarkness2** [comment]
I was looking and the baltop only goes up to 10 if I'm not mistaken, it would be like the other economy plugins that as they enter users go increasing, for example if the maximum baltop today are 20 people and tomorrow I enter 10 more than you when you put /baltop 2 I get from 11 to 20 and so on with the other pages that is how it does for example the essentials and placeholders of top same
